go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/fluxcd/helm-controller/api v1.3.0
 	github.com/giantswarm/apiextensions-application v0.6.2
 	github.com/giantswarm/cluster-standup-teardown v1.36.0
-	github.com/giantswarm/clustertest v1.40.0
+	github.com/giantswarm/clustertest v1.40.1-0.20250925101831-d87e299c2863
```

Comment:
To be changed once this is released https://github.com/giantswarm/clustertest/pull/524